tools/cleanup.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+NAMESPACE="splunk-operator"
+DOMAIN="enterprise.splunk.com"
+
+removeCRDs() {
+	#DOMAIN=$1
+	echo "Removing CRs with domain $DOMAIN"
+	## get list of CRDS
+	echo '$(kubectl api-resources --verbs=list --namespaced -o name | grep -iF "$DOMAIN")'
+	CRDS=$(kubectl api-resources --verbs=list --namespaced -o name | grep -iF "$DOMAIN")
+	for CRD in $CRDS
+	do	
+		echo "In CRD $CRD"
+		## get list of namespaces these crs live
+		NAMESPACES=$(kubectl get $CRD -A -o jsonpath="{.items[*].metadata.namespace}")
+		for NAMESPACE in $NAMESPACES
+		do
+			echo "try to create namespace $NAMESPACE incase no longer exist"
+			kubectl create namespace $NAMESPACE 
```

Comment:
Is this is just for internal usage?
Also, if the namespace is not existing, then there should not be any resource associated with the namespace right? in this case CRDs.

Reply:
this is for internal usage only. this scripts help to cleanup and make cluster clean, I have been using it everyday. create namespace is just to make sure it fails. by the time I selected and i loop, it should be there.